File: test/fixtures/benchmarks/csharp/UntestedModule.cs
// Phase 10i.0.4 — deliberate untested file fixture. No matching
// test file (e.g. UntestedModuleTests.cs) exists; dxkit's
// `test-gaps` filename-match coverage source should report this
// in `gaps[]` with `hasMatchingTest: false`.
namespace Dxkit.Benchmark;

public static class UntestedModule
{
    public static string Describe() => "untested";
}

File: test/fixtures/benchmarks/java/UntestedModule.java
// Phase 10k.1.0 — deliberate untested file fixture, Java row.
//
// No matching test file exists; dxkit's `test-gaps` filename-match
// coverage source should report this in `gaps[]` with
// `hasMatchingTest: false`. Body is intentionally simple: no lint
// violations, no clones, no secrets — those concerns are covered by
// separate matrix fixtures.

public class UntestedModule {
    public String describe() {
        return "untested";
    }
}
